api/src/main/kotlin/edu/wgu/osmt/elasticsearch/ElasticsearchClientManager.kt

```diff
@@ -71,4 +82,36 @@ class ElasticsearchClientManager {
             return UUID.fromString(source)
         }
     }
+
+    private fun createRestClient(): RestClient {
```

Comment:
This seems like library or POC code. Is this needed?

Reply:
Yes, it is to replace the no longer existent RestHighLevelClient in the newer version of ElasticSearch